setup.py

```diff
@@ -1,67 +1,68 @@
-# -*- coding: utf-8 -*-
 from setuptools import find_packages
 from setuptools import setup
 
 
-version = '4.0.1.dev0'
+version = "4.0.1.dev0"
 
-long_description = '{0}\n{1}'.format(
-    open('README.rst').read(),
-    open('CHANGES.rst').read()
+long_description = "{}\n{}".format(
+    open("README.rst").read(), open("CHANGES.rst").read()
 )
 
 extras_require = {
-    'test': [
-        'Products.CMFPlone',
-        'plone.app.testing',
-        'six',
-        'zope.publisher',
+    "test": [
+        "Products.CMFPlone",
+        "plone.app.testing",
+        "plone.testing",
+        "zope.publisher",
     ]
 }
 
 install_requires = [
-    'Acquisition',
-    'Products.GenericSetup',
-    'ZODB',
-    'Zope',
-    'plone.app.vocabularies',
-    'setuptools',
-    'zope.component',
-    'zope.contentprovider',
-    'zope.interface',
-    'zope.site',
-    'zope.viewlet',
+    "Acquisition",
+    "Products.GenericSetup",
+    "ZODB",
+    "Zope",
```

Comment:
shouldnt Zope cover Acquisition, zope.component, zope.interface and ZODB at least?

Reply:
indeed, fixed!